File: typescript/src/numinlist/num_in_list.ts
export const iterate: (list: number[], num: number) => boolean = (
  list,
  num,
) => {
  for (let n of list) {
    if (n === num) {
      return true;
    }
  }
  return false;
};

export const std: (list: number[], num: number) => boolean = (list, num) => {
  return list.includes(num);
};

File: go/src/numinlist/num_in_list.go
package numinlist

// https://github.com/joncalhoun/algorithmswithgo.com/tree/master/module01#05---determine-if-a-number-is-in-a-list-code

// Iterate :
func Iterate(list []int, num int) bool {
	for _, n := range list {
		if n == num {
			return true
		}
	}
	return false
}
